@types/jsb.d.ts

```diff
@@ -461,83 +469,85 @@ declare namespace jsb {
          * @en The base plan id associated with the subscription product.
          * @zh 与订阅产品相关的基本计划 ID。
          */
-        readonly basePlanId: string;
+        getBasePlanId(): string;
         /**
         * @en The offer id associated with the subscription product.
         * @zh 与订阅产品相关的优惠 ID。
         */
-        readonly offerId: string;
+        getOfferId(): string;
         /**
         * @en The offer tags associated with this Subscription Offer.
         * @zh 与此订阅优惠相关的优惠标签。
         */
-        readonly offerTags: string[];
+        getOfferTags(): string[];
         /**
         * @en The offer token required to pass in launchBillingFlow to purchase the subscription product with these pricing phases.
         * @zh 在 launchBillingFlow 中传递以使用这些定价阶段购买订阅产品所需的优惠令牌。
         */
-        readonly offerToken: string;
+        getOfferToken(): string;
         /**
          * @en The pricing phases for the subscription product.
          * @zh 订阅产品的定价区间。
          */
-        readonly pricingPhaseList: PricingPhase[];
+        getPricingPhases(): PricingPhases;
         /**
         * @en The additional details of an installment plan.
         * @zh 分期付款计划的附加详细信息。
         */
-        readonly installmentPlanDetails: InstallmentPlanDetails;
+        getInstallmentPlanDetails(): InstallmentPlanDetails;
     }
     /**
      * @en Represents the details of a one time or subscription product.
      * @zh 代表一次性或订阅产品的详细信息。
      */
-    export interface ProductDetails {
+    export class ProductDetails {
+        static RecurrenceMode: jsb.RecurrenceMode;
```

Comment:
Should use `typeof jsb.RecurrenceMode` for enum type.

Reply:
fixed